packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentInformation.js

```diff
@@ -115,6 +136,27 @@ export const usePaymentInformation = props => {
         onCompleted: onPaymentDetailsQueryCompleted
     });
 
+    const handleExiredPaymentError = useCallback(() => {
```

Comment:
I have tried to reset the `selected_payment_method` on the cart as well, but the GQL server does not allow us to do so. Once set, the `selected_payment_method` can not be changed to `""` which is the default value when a cart is created.

Reply:
Yea I remember this being a thing. Fortunately we just display "done" state based on the completion of the child method and not based on `selected_payment_method`.

Reply:
```suggestion
    const handleExpiredPaymentError = useCallback(() => {
```

Make sure to update all references to this constant.